nuxt-pinia-tailwind/components/RepositoryCard/RepositoryCard.vue

```diff
@@ -41,6 +47,16 @@ export default {
       required: true,
       default: false,
     },
+    meta: {
+      type: Object as PropType<{
+        language: string;
+        updatedAt: Date;
+        forkCount: number;
+        stargazerCount: number;
+      }>,
+      required: true,
+      default: () => {},
```

Comment:
This is actually returning `void` not an actual object.
```suggestion
      default: () => ({
       language: '',
       updatedAt: new Date(),
       forkCount: 0,
       stargazerCount: 0
      }),
```